backend/app/token_whitelists/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .whitelists import (
```

Comment:
All imports in this file are really exports from the `token_whitelists` application, that are useful for other external apps communicating with it. It seems `get_covalent_pair_list` for example doesn't need being exported, etc.

Exporting a function basically means that other apps can start depending on this given function. We should thus strive to export the minimum.

(I'm using the "application" term as a reference to hexagonal apps: each of `treasury`, `spread` and `token_whitelists` are applications, or maybe "app-modules")